HomeWork03/task02.py

```diff
@@ -0,0 +1,16 @@
+# 2. Напишите программу, которая найдёт произведение пар чисел списка.
+#  Парой считаем первый и последний элемент, второй и предпоследний и т.д.
+import random
+n = int(input('Введите количество элементов исходного списка: '))
+lst = [random.randint(-10, 10) for i in range(n)]
+print(lst)
+result_lst = []
+if n % 2 == 0:
+    res_size = n//2
+else:
+    res_size = n//2+1
+for i in range(res_size):
```

Comment:
Если подумать, то можно заменить, что вне зависимости от того, четная или нечетная длина массива, правильно будет всегда так:
```suggestion
for i in range((n + 1) // 2):
```